pkg/metrics/metrics.go

```diff
@@ -157,7 +159,7 @@ func (m *Metrics) Start(listenAddress string) (*http.Server, error) {
 
 	go func() {
 		log := m.log.WithValues("address", ln.Addr())
-		log.Info("listening for connections on")
+		log.V(logf.InfoLevel).Info("listening for connections on")
```

Comment:
This seems a bit weird. The address will be added to every log message in this function. I think it should only be added to the "listening" message.

Reply:
can be useful for the error being isolated imo